src/core/hle/D3D8/Direct3D9/FixedFunctionPixelShader.hlsli

```diff
@@ -63,10 +63,16 @@ namespace FixedFunctionPixelShader {
 	const float X_D3DTA_COMPLEMENT = 0x00000010;  // take 1.0 - x (read modifier)
 	const float X_D3DTA_ALPHAREPLICATE = 0x00000020;  // replicate alpha to color components (read modifier)
 
-	const int SAMPLE_NONE = 0;
-	const int SAMPLE_2D = 1;
-	const int SAMPLE_3D = 2;
-	const int SAMPLE_CUBE = 3;
+    const int SAMPLE_NONE = 0;
```

Comment:
Indentation is different from surrounding lines

Reply:
not seeing this, it is 4 space indent above and below this line and everything looks normal in VS (white space shows the same), originally could have been tab I suppose..I will tab it and upload